comet/comet_test.go

```diff
@@ -116,13 +125,24 @@ func TestStatus(t *testing.T) {
 }
 
 func TestBroadcastTx(t *testing.T) {
-	chainID := "0"
-	app := testapp.NewTest(t, chainID)
+	app := testapp.NewTest(t, chainID.String())
+	_, err := app.InitChain(context.Background(), &abcitypes.RequestInitChain{
+		ChainId: chainID.String(),
+		AppStateBytes: func() []byte {
+			appStateBytes, err := json.Marshal(testapp.MakeGenesisAppState(t, app))
+			require.NoError(t, err)
+			return appStateBytes
+		}(),
+	})
+	require.NoError(t, err)
+
+	ctx := app.GetContext(true)
+	sk, _, acc := app.TestAccount(ctx)
```

Comment:
This is a common pattern across every test that needs to sign transactions with testapp.ToTx. We need to call InitChain before we fetch the application context, and then we get our pre-funded test account with app.TestAccount.

It'd probably be good to encapsulate this setup logic in a helper function (perhaps inside of testapp.NewTest itself), so we can avoid repeating it across each test.